src/sedcli_main.c

```diff
@@ -974,6 +1008,25 @@ static int handle_write_mbr(void)
 	return ret;
 }
 
+static int handle_blocksid(void)
+{
+	struct sed_device *dev = NULL;
+	int ret = sed_init(&dev, opts->dev_path);
+
+	if (ret) {
+		sedcli_printf(LOG_ERR, "Error in initializing the dev: %s\n",
+				opts->dev_path);
+		return ret;
+	}
+
+	ret = sed_issue_blocksid_cmd(dev, opts->hardware_reset);
```

Comment:
do we need to make check if block sid feature is supported by tper before actually calling it?

Reply:
this is a tricky one that I think the tool shouldn't get too smart about that to make such decision for user. In other words, the user should check level0 discovery to see whether BlockSID feature descriptor is present in order to issue blockSID cmd. If drive doesn't support it, it should just drop or return error.

Reply:
can you please explain why it is tricky?

Reply:
I don't mean adding code to check block sid availability is tricky, but this can introduce issue such as drive may have bug in level0 discovery where it didn't expose block sid availability correctly. In this case, sedcli may incorrectly determine blocksid will not work on the drive, hence allowing user to checkout the feature.